code/datums/components/itemblock/saberblock.dm

```diff
@@ -1,23 +1,27 @@
 //adds object properties to the sword while blocking with it.
 datum/component/itemblock/saberblock
 	bonus = 1 //bonus is a flag that determines whether or not the item tooltip will include "⛨ Block+: RESIST with this item for more info" when not blocking
+	var/can_block
```

Comment:
can_block reads as a name of a boolean variable to me. Maybe `can_block_check` or `can_block_getter` or something like that?

Reply:
Yup, that's a good point!